examples/logger.py

```diff
@@ -3,13 +3,33 @@
 """
 
 import time
+import logging
 from typing import Any, Optional, Tuple
 
 from PyQt5.QtCore import QObject, pyqtSlot, pyqtSignal
 from PyQt5.QtWidgets import QWidget, QVBoxLayout, QPushButton, QTextEdit, QLabel, QDialogButtonBox
 
 from qiwis import BaseApp
 
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+class LoggingHandler(QObject, logging.Handler):
```

Comment:
Of course you might want to make the code simpler by merging the `Signaller` in the example with the handler.
However, multiple inheritance could bring us subtle problems such as confusing method name resolution, and `super()` calls.
I would recommend to separate the `QObject` class and `logging.Handler` class, just like the example.

Reply:
I also agree with @kangz12345.

Reply:
@Aijuh Please modify this part!